observabilityclient/v1/base.py

```diff
@@ -0,0 +1,111 @@
+#   Copyright 2022 Red Hat, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+
+import os
+import shutil
+
+from osc_lib.command import command
+from osc_lib.i18n import _
+
+from observabilityclient.utils import runner
+from observabilityclient.utils import shell
+
+
+OBSLIBDIR = shell.file_check('/usr/share/osp-observability', 'directory')
+OBSWRKDIR = shell.file_check(
+    os.path.expanduser('~/.osp-observability', 'directory')
```

Comment:
I tried the PR and got:
```
Could not load 'observability_discover': expanduser() takes 1 positional argument but 2 were given
```
when trying to do `openstack observability discover`. Looks to me, like it's caused by this line, shouldn't it be changed to the following?
```
os.path.expanduser('~/.osp-observability'), 'directory'
```

Reply:
That you for running the code, @vyzigold ! :)

Reply:
Yeah you're right. My bad.

Reply:
I pushed the patch myself when I encountered the problem again earlier this week.